lib/Plugin.class.php

```diff
@@ -53,6 +53,7 @@ public function instantiateCompiler()
         $this->compiler = new WPLessCompiler;
         $this->compiler->setVariable('stylesheet_directory_uri', "'" . get_stylesheet_directory_uri() . "'");
         $this->compiler->setVariable('template_directory_uri', "'" . get_template_directory_uri() . "'");
+        if(!WP_DEBUG) { $this->compiler->setFormatter('compressed'); }
```

Comment:
- I have fixed not defined WP_DEBUG
- There is no way how to know if formatter already was set
https://github.com/oyejorge/less.php/blob/master/lessc.inc.php#L38